lib/streams.gi

```diff
@@ -234,6 +234,31 @@ function( stream )
     CloseStream(stream);
 end );
 
+BindGlobal("EvaluateString",
```

Comment:
Not a change request per se, just rambling: I am somewhat concerned about having an undocumented function `EvaluateString` next to the documented `EvalString`, as I am pretty sure people will discover the undocumented one quickly from tab completion, and there will be confusion about this.

For `LIBGAP_EvalString`, the name makes it clear what it is for. Not so for, which has zero documentation, and also is not mentioned in the commit message nor the description of this PR... So what goes?